oak_kernel_measurement/README.md

```diff
@@ -0,0 +1,18 @@
+# Kernel Measurement Prediction Tool
+
+When booting a VM the kernel and its associated setup data is measured by the
+Stage 0 firmware. The VMM accepts a single bzImage format kernel file. It then
```

Comment:
VMM = Virtual Machine Manager (just looked it up). Probably fine to leave if it is commonplace.